float32     dis_to_obj # distance to the object
float32     steer # steering angle
float32     coll_prob # collision probability